diff_general/run_autograder

```diff
@@ -4,4 +4,4 @@ cd /autograder/source
 
 bash ./compile.sh
 
-python run_tests.py > /autograder/results/results.json
+python3 run_tests.py
```

Comment:
this is a nice change, since I've seen this go wrong a few times when also using the `stream` kwarg in `JSONTestRunner`